src/extension.ts

```diff
@@ -54,17 +54,20 @@ export function activate(context: vscode.ExtensionContext) {
     getDotENVS();
     suggestEnvVariables = suggestENVSFromDotEnv();
   } else {
```

Comment:
Need a comment above this `if/else` explaining *why* we need two types of suggestion mechanism

Reply:
Added it